lib/reply.js

```diff
@@ -88,10 +88,20 @@ Reply.prototype.hasHeader = function (key) {
   return this._headers[key.toLowerCase()] !== undefined
 }
 
+Reply.prototype.removeHeader = function (key) {
+  // Node.js does not like headers with keys set to undefined,
```

Comment:
Still valid?

Reply:
yes, one might argue why we don't do `this._headers[key] = undefined`.

Reply:
ahh, ok.